TempoCore/Source/TempoCore/Public/TempoPlayerController.h

```diff
@@ -1,28 +1,174 @@
-﻿// Copyright Tempo Simulation, LLC. All Rights Reserved
+// Copyright Tempo Simulation, LLC. All Rights Reserved
 
 #pragma once
 
 #include "CoreMinimal.h"
 #include "GameFramework/PlayerController.h"
+#include "Components/SpotLightComponent.h"
 #include "TempoPlayerController.generated.h"
 
+class ASpectatorPawn;
+class UUserWidget; // Forward declaration for UUserWidget class
```

Comment:
```suggestion
class UUserWidget;
```
Comments like this are just clutter. A C++ engineer will recognize a forward declaration. Or if they don't, they should look it up and learn. It's no more your job to explain the features you use than it is a novelist's job to define the words they use.